onnxruntime/test/python/onnxruntime_test_python.py

```diff
@@ -573,7 +573,9 @@ def testSequenceConstruct(self):
         np.testing.assert_array_equal(output_expected, res[0])
 
     def testSequenceInsert(self):
-        sess = onnxrt.InferenceSession(self.get_name("sequence_insert.onnx"))
+        opt = onnxrt.SessionOptions()
+        opt.execution_mode = onnxrt.ExecutionMode.SEQUENTIAL
```

Comment:
previous version of your test was failing because for the python bindings if it was built with CUDA, it will be registered by default during session creation. if you enable parallel execution as well, it will fail.
you can add another check prior to creating the session. 
```
opt = onnxrt.SessionOptions()
opt.execution_mode = onnxrt.ExecutionMode.PARALLEL
if not 'CUDAExecutionProvider' in onnxrt.get_available_providers():
    sess = onnxrt.InferenceSession(..., sess_options=opt)
```

Reply:
Got it. Thanks!